File: app/src/main/java/ds/photosight/ui/TopsFragment.java
package ds.photosight.ui;


import ds.photosight.Constants;
import ds.photosight.R;

public class TopsFragment extends ListFragmentAbstract {

	@Override
	protected ListAdapter initList() {
		return new ListAdapter(getActivity(), R.layout.simple_list_item_activated_1, getResources().getStringArray(R.array.tops_array));
	}


	@Override
	protected int getListType() {
		return Constants.TAB_TOPS;
	}
}

File: app/src/main/java/ds/photosight/ui/TopsFragment.kt
package ds.photosight.ui


import ds.photosight.Constants
import ds.photosight.R

class TopsFragment : ListFragmentAbstract() {

    override fun initList(): ListAdapter {
        return ListAdapter(activity!!, R.layout.simple_list_item_activated_1, resources.getStringArray(R.array.tops_array))
    }

    override val listType: Int
        get() = Constants.TAB_TOPS
}

